remote_monitoring_demo/demo.py

```diff
@@ -130,13 +142,39 @@ def main(argv):
     print("Updating results for one of the experiments and fetching recommendations from Kruize...")
     print("*************************************************************************************\n")
 
+    bulk_payload = []
     for i in range(1, num_exp_res):
-        json_file = "./resource_usage_metrics_data/result_" + str(i) + ".json"
-        update_results(json_file)
-
-        reco = list_recommendations(experiment_name)
-        recommendations_json_arr.append(reco)
-
+        json_file_path = "./resource_usage_metrics_data/result_" + str(i) + ".json"
+        with open(json_file_path, 'r') as json_file:
+            json_data = json_file.read()
+        json_parsed = json.loads(json_data)
+        bulk_payload.append(json_parsed[0])
+
+    # Define the batch size
+    batch_size = 100
+
+    # Loop to fetch elements in batches
+    current_index = 0
+    while current_index < len(bulk_payload):
+        # Get the current batch
+        batch = bulk_payload[current_index:current_index + batch_size]
+
+        file_path = './resource_usage_metrics_data/result_%s_to_%s.json' % (current_index, batch_size)
+        with open(file_path, 'w') as json_file:
+            json.dump(batch, json_file)
+        update_results(file_path)
+        batch_deep_copy = copy.deepcopy(batch)
+        for item in batch_deep_copy:
+            item['interval_start_time'] = datetime.strptime(item['interval_start_time'], "%Y-%m-%dT%H:%M:%S.%fZ")
+            item['interval_end_time'] = datetime.strptime(item['interval_end_time'], "%Y-%m-%dT%H:%M:%S.%fZ")
+        max_time = max(batch_deep_copy, key=lambda x: x['interval_end_time'])['interval_end_time']
+        update_recommendations(experiment_name, max_time.strftime("%Y-%m-%dT%H:%M:%S.%fZ")[:-4] + "Z")
```

Comment:
Can we suppress printing the recommendation JSON that is returned from this call. It adds too much info to the demo output.

Reply:
done